supported-platforms.md

```diff
@@ -0,0 +1,20 @@
+# Supported Platforms
+
+A Kubernetes release consists of a set of Go binaries for various Operating Systems and CPU architectures. Platforms are identified using the Go `<GOOS>-<GOARCH>` parlance.
```

Comment:
```suggestion
A Kubernetes release consists of a set of Go binaries and container images for various Operating Systems and CPU architectures. Platforms are identified using the Go `<GOOS>-<GOARCH>` parlance.
```

Reply:
There are also various scripts and manifests, looking at the contents of e.g. https://github.com/kubernetes/kubernetes/releases/download/v1.18.12/kubernetes.tar.gz

Reply:
@spiffxp yep, this is not meant to cover _every artifact_ produced, just the supported platforms. That being said, I do think we should document those as well.